RELEASE.md

```diff
@@ -58,37 +58,51 @@ The Release Manager must create a release candidate branch:
     git checkout -b 'release-x.x.x'
     ```
 
--   [ ] 4.  Update Dependencies
+-   [ ] 4.  Tag the commit with a release candidate version following the pattern `x.x.x-RC-0`
+
+    For example:
+
+    ```bash
+    git tag 'x.x.x-RC-1'
+    ```
+
+    This is a requirement so that we can fill in values for `engine` and `documentation` versions on the next step.
+
+-   [ ] 5.  Update Dependencies
 
     Let's assume that `requirements.yaml` file should have the correct dynamic versions configured.
 
     Run the following command:
 
     ```bash
-    make update_dependencies
+    make create_version_file update_dependencies
```

Comment:
Here we create the version file first and then are free to update dependencies and versions on the values files.